{{cookiecutter.project_slug}}/frontend/app/content/about.md

```diff
@@ -35,7 +35,7 @@ This project is a comprehensively updated React + MongoDB version of [Sebastián
   - [**Many other features**]("https://fastapi.tiangolo.com/features/"): including automatic validation, serialization, interactive documentation, etc.
 - [**Next/React**](https://react.dev/) frontend:
   - **Authentication** with JWT and cookie management, including `access` and `refresh` tokens,
-  - **Authorisation** via middleware for page access, including logged in or superuser.
+  - **Authorization** via middleware for page access, including logged in or superuser.
```

Comment:
Same change needed as in the above `Authorization` section.

Reply:
Removed the change here and will explicitly resolve in #14